### ISSUE TITLE
Rest diypunks inscription data and diypunk-v2 data

### ISSUE THREAD

**Issue by NiceKey8280** [ordbase/generative-orc-721#6]
We care about the rest of the diypunks list. 
Some guys mint diypunks after 721 , because he found some broken inscriptions. 
We want to know range of diypunks collection
I think its fcfs for diypunks holder.

**Comment by geraldb** [comment]
> I think its fcfs [first-come / first-serve] for diypunks holder [minter].

yes, the ORC-721 protocol rule / principle is "first-come / first-serve"  BUT the mint inscription MUST be valid to count - to be valid the mint text inscription 

1. MUST be valid json  - try here -> https://jsonlint.com - and 
2.  the g specs / numbers MUST be all in range e.g. 0-59 (60) and 
3. no more than one mint in the inscription and 
4.  the protocol keys must match e.g. p / op / slug . 

That's it.     Thank you for you patience and undertanding in getting the validation automated so everyone can run / validate (any ORC-721 collection.

**Comment by terrific188** [comment]
Awesome!

**Comment by geraldb** [comment]
@NiceKey8280   now that things are slowing down in ordgen / orc-721.   the diy punks mint list is here -> <https://github.com/ordbase/generative-orc-721/blob/master/diypunks/mint.csv> and diy punks v2 mint list is here -> https://github.com/ordbase/generative-orc-721/blob/master/diypunks-v2/mint.csv

does this answer your questions from back in june?
